doc/api/events.md

```diff
@@ -1819,6 +1819,22 @@ added: v14.5.0
 
 This is not used in Node.js and is provided purely for completeness.
 
+#### `event.initEvent(type[, bubbles, cancelable])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 3 - Legacy
+
+* `type` {string}
+* `bubbles` {boolean}
+* `cancelable` {boolean}
+
+Redundant with event constructors and incapable of setting `composed`.
+This is not used in Node.js and is provided purely for completeness.
+The WHATWG spec considers it deprecated and users shouldn't use it at all.
```

Comment:
nit: This should probably go in the `Legacy` banner instead?

Reply:
moved this sentence to `Legacy` banner.